iOS/FlowWalletKit/Sources/Storage/KeychainStorage.swift

```diff
@@ -25,11 +25,14 @@ public class KeychainStorage: StorageProtocol {
                 .label(label)
                 .accessibility(.afterFirstUnlock)
                 .synchronizable(synchronizable)
+                .accessibility(.whenUnlockedThisDeviceOnly)
+                
         } else {
             keychain = Keychain(service: service)
                 .label(label)
                 .synchronizable(synchronizable)
                 .accessibility(.afterFirstUnlock)
+                .accessibility(.whenUnlockedThisDeviceOnly)
```

Comment:
Same here

Reply:
update to `.accessibility( deviceOnly ? .whenUnlockedThisDeviceOnly : .whenUnlocked)`